app/src/main/java/com/br/imed_app2/model/Drug.kt

```diff
@@ -0,0 +1,18 @@
+package com.br.imed_app2.model
+
+import java.time.LocalDateTime
+
+class Drug (
```

Comment:
Please auto-format this file,

Reply:
Done 7b19a0b5ebbbd1688941b012dc10cf481ed5f0ae